.travis.yml

```diff
@@ -1,8 +1,11 @@
-language: generic
+language: node_js
+node_js:
+  - "10"
 
 env:
   - OCAML=4.02.3
   - OCAML=4.02.3+buckle-master
+  - OCAML=4.02.3+buckle-master ESY_BUILD=YES ESY__CACHE=/home/travis/.esy
```

Comment:
Can you help me understand what this accomplishes?

Reply:
We need to build `odoc` using this version of the compiler to be able to read `.cmti` files compiled with it.

Reply:
@ostera AFAIK esy will use its own internal oap switch to build the `odoc` binary.

Reply:
@aantron @jordwalke I left that in there because I was installing opam/ocaml even for the esy build. I realize now that it's not needed at all, so I will remove it and make the installation optional.